notifier/alertmanagerset_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2013 The Prometheus Authors
```

Comment:
We're leaving out dates from new files.

```suggestion
// Copyright The Prometheus Authors
```